dwave/cloud/cli.py

```diff
@@ -904,15 +904,26 @@ def _install_contrib_package(name, verbose=0, prompt=True):
 @click.option('--install-all', '--all', '-a', default=False, is_flag=True,
               help='Install all non-open-source packages '\
                    'available and accept licenses without prompting')
+@click.option('--auth', default=False, is_flag=True,
+              help="Authorize Ocean to access Leap API on user's behalf. "
+              "Implies --auto-token during 'dwave config create' and it's "
+              "mutually exclusive with --full.")
```

Comment:
Perhaps something like "Includes running `dwave config create --auto-token`. Cannot be used together with the `--full` option."?